perfkitbenchmarker/linux_packages/collectd.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Builds collectd from source, installs to VM_TMP_DIR.
+"""Builds collectd from source, installs to /opt/pkb.
```

Comment:
Maybe just say `installs to INSTALL_DIR` so that we won't have to change it again